packages/oats-runtime/src/reflection-type.ts

```diff
@@ -104,6 +104,7 @@ export interface Props {
 }
 
 export type AdditionalProp = boolean | Type;
+export type WebFile = File;
```

Comment:
I don't know for sure. But would like to try it out

Reply:
I am following up as @syedtauhidahmed is out of office for rest of the week

@sakari - do you know if there is a way to release an alpha for example so that we can test if this will fix the issue we are seeing?

I don't have all the background info about why this is needed btw but I'm a bit blocked by this at the moment so I am interested to see if this could be a fix ☺️

Reply:
File (the web api kind) is [available after node 20](https://github.com/colinhacks/zod/issues/387#issuecomment-1774603011) which I think not everyone in the world is using yet. That being said I published a canary package version 7.4.8-alpha.0 from this branch for you to test.

Reply:
Rewrote the WebFile as:

```
export interface WebFile extends Blob {
  readonly lastModified: number;
  readonly name: string;
  readonly webkitRelativePath: string;
};
```
`Blob` class is a global reference for require('node:buffer').Blob --> since v18

Reply:
unfortunately the alpha version did not have any effect the request still fails